web/static/aws/connect/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
Your HTML isn't auto-formatted - please use Prettier as your HTML formatter in VS Code.

Reply:
it is according to my formatter